pkg/pkg.bzl

```diff
@@ -379,3 +379,57 @@ def pkg_deb(name, package, **kwargs):
         changes = out_changes,
         **kwargs
     )
+
+def _pkg_zip_impl(ctx):
+    args = ctx.actions.args()
+
+    for f in ctx.files.srcs:
+        args.add("-f", "%s=%s" % (_quote(f.path), dest_path(f, strip_prefix = None)))
+
+    args.add("-o", ctx.outputs.out.path)
+    args.add("-d", ctx.attr.package_dir)
+    args.add("-t", ctx.attr.timestamp)
+    args.use_param_file("%s")
```

Comment:
As written, this does not work if there are a large number of files. If it does spill args to a file build_zip will get invoked with just the name of the temp file. And then it will be sad....  

If we were using absl flags, then typically the format would be 
   args.use_param_file("--flagfile=%s")
because absl flags treats flagfile specially (see: https://abseil.io/docs/python/guides/flags)

But argparse does not do this magically, it uses fromfile_prefix_chars to do indirection or args to a file. So
     args.set_param_file_format("multiline")
     args.use_param_file("@%s")
FWIW, use_param_file is underdocumented. It needs an example of how it must actually be used.

Reply:
Yeoch, sorry. I forgot I had configured this!